2.1.1/Dockerfile.build

```diff
@@ -0,0 +1,128 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
```

Comment:
I am concerned about this file falling out of sync with the main `Dockerfile`. Is it possible to generate all images from the same file?

Reply:
I took care of it. Basically now everything should get done via one file only.